### ISSUE TITLE
Flaky Test Case in RPN Label Encoder and ROI Sampler

### ISSUE THREAD

**Issue by ariG23498** [keras-team/keras-cv#2336]
### Description

There's a flaky test case in the `rpn_label_encoder_test.py`, specifically linked [here](https://github.com/keras-team/keras-cv/blob/94d7327735a2921b4da37939a5d61cd6fb9b525a/keras_cv/layers/object_detection/rpn_label_encoder_test.py#L70). The flakiness appears to be due to the use of the random API, which introduces non-deterministic behavior in test outcomes.

### Problem Demonstration

A deeper investigation into the issue is documented in this [Colab Gist](https://colab.research.google.com/gist/ariG23498/e8322c221b3c1ed666a7cf4af5f83f80/scratchpad.ipynb), which outlines how the random API's usage is leading to inconsistent test results.

CC: @divyashreepathihalli

**Comment by ariG23498** [comment]
Adding to the previously stated test failing, I think [`test_roi_sampler_small_threshold`](https://github.com/keras-team/keras-cv/blob/3d0d1ddc3069446f49b877093c0b466a3fec87a0/keras_cv/layers/object_detection/roi_sampler_test.py#L70) has a similar behavoiur.

Reproduction of the error: [GitHub Gist](https://colab.research.google.com/gist/ariG23498/f0325ee179597f66f52a5ce96f83cea6/scratchpad.ipynb)

**Comment by sachinprasadhs** [comment]
Thanks for reporting the issue! We have consolidated the development of KerasCV into the new **KerasHub** package, which supports image, text, and multi-modal models. Please read [the announcement](https://github.com/keras-team/keras-hub/issues/1831). `KerasHub` will support all the core functionality of `KerasCV`.

`KerasHub` can be installed with `!pip install -U keras-hub`. Documentation and guides are available at [keras.io/keras_hub](https://keras.io/keras_hub/).

With our focus shifted to KerasHub, we are not planning any further development or releases in KerasCV. If you encounter a KerasCV feature that is missing from KerasHub, or would like to propose an addition to the library, please file [an issue with KerasHub](https://github.com/keras-team/keras-hub/issues).
